lib/slack_transformer/html/lists.rb

```diff
@@ -21,19 +21,18 @@ def to_slack
           end
         end
 
-        fragment.to_html
+        fragment.to_html(save_with: 0)
```

Comment:
The value 0 for save_with is the integer representative of which one of [these options](https://nokogiri.org/rdoc/Nokogiri/XML/Node/SaveOptions.html)? Could you replace the number with its equivalent enum?

Reply:
Jus add a short comment explaining what was happening and how `(save_with: 0)` saves us from it :)